### ISSUE TITLE
Shell script for updating to latest template version

### ISSUE THREAD

**Issue by ghost** [schnes4/dhbw-heidenheim-latex-template#16]
A shell script is needed to update your local version with a newer version from the template. The shell script handles the **unwanted** merge conflicts possibly occuring when updating to a newer template version.

**Comment by StefanFW** [comment]
Please keep in mind that you have to exclude the following files:

- content/*
- images/*
- bibliographie.bib
- readme.md

**Comment by nobodypb** [comment]
All files that have to be changed in order to create the actual paper should reside in one subdirectory. We could then remove that directory from the git index after cloning and create a new submodule for the actual work. This would allow updating of the template specific files, while allowing an own vcs for the actual work. Note, that all files in that directory should have at least dependencies to the template as possible to preserve backward compatibility.

**Comment by StefanFW** [comment]
I don't think that we should use git submodules. Please keep in mind that not every user is familiar with deeper git concepts like submodules or even worser with git as itself. I'm personal not a friend of them and I don't think that they are necessary in this context. 

Also I don't agree with the subdirectory approach! This would mean that we sacrifice structure for "easier" updating.

Did you check the actual state of the update script?
@kiessla could you please reference this Issue in your commits?
@kiessla please give us a short overview of the actual state so we could help you implementing this feature.

**Comment by nobodypb** [comment]
I never meant using submodules in this project. My intention was to restructure the project in order to make using a submodule for the actual paper possible. 
Well this project is meant as template and as such the structuring, template specific parts should be separated from actual content parts in order to keep updateability and to decrease the probability of user made errors. 
I think we should comply with the usability regarding the enduser instead of a pure logical structure.

**Comment by StefanFW** [comment]
mhm…
could you please give an example for the structure as you would design it?
